testutil/network.go

```diff
@@ -14,29 +14,23 @@ import (
 )
 
 func checkPortIsOpen(port int) bool {
-	timeout := time.Millisecond * 10
+	timeout := time.Second
```

Comment:
This is wrong - a millisecond is a thousandth (0.001 or 10−3 or 1/1000) of a second.